3scale/APIcast
# Remove s2i

### what

Fixes: https://issues.redhat.com/browse/THREESCALE-8132

### Verification steps

Make sure s2i is not installed in the system

* build runtime image
```
make runtime-image
....
Successfully tagged apicast-runtime-image:latest
```

* test runtime image just built
```
make test-runtime-image IMAGE_NAME=apicast-runtime-image:latest
```

Depends on #1335
